docs/migration_guide.md

```diff
@@ -45,13 +45,17 @@ When upgrading your existing Habor instance to a newer version, you may need to
 
 7. Unzip the new Harbor package and change to `./harbor` as the working directory. Configure Harbor by modifying the file `harbor.cfg`, 
 
-7.1 Configure Harbor by modifying the file `harbor.cfg`, 
+  - Configure Harbor by modifying the file `harbor.cfg`, 
 you may need to refer to the configuration files you've backed up during step 2. 
 Refer to [Installation & Configuration Guide ](../docs/installation_guide.md) for more information. 
 Since the content and format of `harbor.cfg` may have been changed in the new release, **DO NOT directly copy `harbor.cfg` from previous version of Harbor.** 
 
-7.2 Migrate 'harbor.cfg' with upgrade script,
-Under the directory `./harbor`, run the `./upgrade --source_loc source_harbor_cfg_loc --source_version 0.5.0 --target_loc target_harbor_cfg_loc --target_version 1.1.0` to upgrade harbor.cfg to 1.1.0. This script only support upgrade from 0.5.0 to 1.1.0.
+  - To assist you in migrating the harbor.cfg file from v0.5.0 to v1.1.0, a script is provided and described as below. For other versions of Harbor, you need to manually migrate the file harbor.cfg.
```

Comment:
Add this at the end:
Note: After running the script, make sure you go through harbor.cfg to verify all the settings are correct. You can make changes to harbor.cfg as needed.

^^^^ this paragraph, quote  harbor.cfg .